README.md

```diff
@@ -281,3 +281,33 @@ The test doesn't require any special setup. The instructions to run it are insid
 [utam-js]: https://www.github.com/salesforce/utam-js
 [utam-doc]: https://utam.dev
 [utam-java-recipes]: https://www.github.com/salesforce/utam-java-recipes
+
+## Run Salesforce Mobile test
+
+- Follow the instructions at [Get Started for Mobile](https://utam.dev/guide/get_started_utam#get-started-for-mobile) to set up your local simulator/emulator.
+- Make sure [Appium](https://github.com/appium/appium#readme) and [node.js](https://nodejs.org/en/) are installed on your machine.
+- Update the wdio configuration file:
+For an iOS test, update `wdio.conf.ios.js` file to configure, test device name and os version, the full path for the test application:
```

Comment:
```suggestion
For an iOS test, update `wdio.conf.ios.js` file to configure the test device name, iOS version, and the full path for the test application:
```

Reply:
I'm not clear which of the properties listed above correspond to the example below so it would be good to list them in the same order in the text intro and the js block below. something like:
For an iOS test, update `wdio.conf.ios.js` file to configure the test device name (`appium:deviceName`), the full path for the test application (`appium:app`), and the iOS version (`appium:platformVersion` TODO: not sure if I'm listing the correct prop here so please confirm):